README.md

```diff
@@ -133,6 +142,13 @@ to a customized request topic name. The virtual asset deployment should be confi
 3. Start PDA and change queue type to Kafka in **PARASOFT QUEUE CONFIGURATION** section of PDA Demo Administration page.
 4. To test connection with Kafka server, either use **Test Connection** button in **Kafka configuration details** link or save changes in PDA Demo Administration page.
 
+### Using external RabbitMQ server with PDA
+1. Download and install Erlang.
+2. Download, install and start a RabbitMQ server (RabbitMQ version should correspond to Erlang RabbitMQ) using default settings.
```

Comment:
1. Please test the function in PDA with some lower version of RabbitMQ server.
2. Change the sentence in parenthesis to "(RabbitMQ version should be compatible with Erlang version)".